packages/root-cms/ui/components/DocEditor/DocEditor.tsx

```diff
@@ -1169,6 +1169,7 @@ DocEditor.OneOfField = (props: FieldProps) => {
           onChange={(e: string) => onTypeChange(e || '')}
           size="xs"
           radius={0}
+          searchable
           // Due to issues with preact/compat, use a div for the dropdown el.
           dropdownComponent="div"
```

Comment:
were you able to test this change? reason i ask is because the `dropdownComponent="div"` was added because of some incompatibility issues and so i wanna make sure this works with the div implementation.

Reply:
Yeah; I gave it a try on the `docs` package and it LGTM. It's a little quirky in that it supports substring matches but that actually makes it easier to locate the desired value